profiles/ritika1.md

```diff
@@ -0,0 +1,15 @@
+## Ritika Varshney
+
+### Bio
+I am interseted in Machine learning and web development.
+
+### Location
+Bareilly
+
+
+### Hobbies & Interests
+Listen to music ,cooking.
+
+### Profiles
+[![GitHub](https://github.com/ritikavar) 
+  
```

Comment:
Add these 2 lines after this:

`[twitter-img]: https://i.imgur.com/wWzX9uB.png`
`[github-img]: https://i.imgur.com/9I6NRUm.png`